apps/ledger-live-mobile/src/newArch/features/Web3Hub/components/Disclaimer/useDisclaimerViewModel.ts

```diff
@@ -71,6 +76,9 @@ export default function useDisclaimerViewModel(goToApp: (manifestId: string) =>
   }, [disclaimerManifest, goToApp, isChecked, setWeb3HubDB]);
 
   const onClose = useCallback(() => {
+    track("web3hub_disclaimer_closed", {
+      component: `Disclaimer`,
+    });
```

Comment:
That might send too much informations because close is usually called multiple times but also bad informations because close is called regardless of it being close onConfirm or when really closing